### PR TITLE
AMBR-316 : Initial work to increase code coverage.

### PR DESCRIPTION
**Pull request by randy-tano**

### CODE REVIEW

File: src/main/java/org/ambraproject/wombat/controller/ArticleController.java [hunk 1]
```diff
@@ -864,19 +865,6 @@ private XmlContent getXmlContent(Site site, ArticlePointer articlePointer,
     });
   }
 
-  private String getLinkText(Site site, HttpServletRequest request, String doi) throws IOException {
```
Comment (sgatchell) [inline src/main/java/org/ambraproject/wombat/controller/ArticleController.java]: always nice to remove dead code

File: src/test/java/org/ambraproject/wombat/util/ThemeTest.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+import com.google.common.collect.ImmutableMap;
+
+import freemarker.cache.TemplateLoader;
+
+/**
+ * Class to support unit tests.  The class returns <b>null</b> from all the methods,
+ * so this class <b>MUST</b> be mocked, and the mocking instance must provide all
+ * the necessary returns.
+ */
+public class ThemeTest extends Theme {
```
Comment (sgatchell) [inline src/test/java/org/ambraproject/wombat/util/ThemeTest.java]: I this for future use?  Tests still pass when I remove it.

File: pom.xml [hunk 3]
```diff
@@ -274,9 +275,18 @@
     </dependency>
     <dependency>
       <groupId>org.mockito</groupId>
-      <artifactId>mockito-all</artifactId>
-      <version>1.9.5</version>
-      <scope>test</scope>
+      <artifactId>mockito-core</artifactId>
+      <version>2.3.0</version>
```
[the remark this answers is not in the document]
Comment (randy-tano) [reply pom.xml]: Good house keeping, to get the latest version.

File: src/main/java/org/ambraproject/wombat/controller/ArticleController.java (hunk 1 above)
Comment (randy-tano) [reply src/main/java/org/ambraproject/wombat/controller/ArticleController.java]: Yeah, quick win to improve code coverage ;)

**Pull request merged**